Makefile

```diff
@@ -166,8 +165,7 @@ core_integration_test: core_integration_server_start core_integration_test_packa
 
 .PHONY: console_lint
 console_lint:
-	cd console && \
-	yarn lint
+	pnpm turbo lint --filter=./console
```

Comment:
On a new Zitadel install, this won't unless you knowingly call `make console_dependencies` first.

Shouldn't change this to:

```
console_lint: console_dependencies
  pnpm turbo lint --filter=./console
```

?